mlflow/utils/environment.py

```diff
@@ -193,7 +193,8 @@ def _is_iterable(x):
 
 
 _INFER_PIP_REQUIREMENTS_FALLBACK_MESSAGE = (
-    "Encountered an unexpected error while inferring pip requirements (model URI: %s, flavor: %s)"
+    "Encountered an unexpected error while inferring pip requirements (model URI: %s, flavor: %s)."
+    " To disable the auto dependency inference, specify `conda_env` or `pip_requirements`."
 )
```

Comment:
The new message allows users to quickly suppress an error stack trace for the inference failure.